satpy/tests/test_modifiers.py

```diff
@@ -593,8 +599,18 @@ def test_get_angles_satpos_preference(self, forced_preference):
             (_diff_sat_pos_datetime, False, 6),
         ]
     )
-    @pytest.mark.parametrize("input_func", [_get_angle_test_data, _get_stacked_angle_test_data])
-    def test_cache_get_angles(self, input_func, input2_func, exp_equal_sun, exp_num_zarr, force_bad_glob, tmpdir):
+    @pytest.mark.parametrize(
+        ("input_func", "num_normalized_chunks"),
+        [
+            (_get_angle_test_data, 9),
+            (_get_stacked_angle_test_data, 3),
+            (_get_angle_test_data_odd_chunks, 9),
+        ])
+    def test_cache_get_angles(
```

Comment:
This function is starting to get long, can it be split somehow?

Reply:
CodeScene was fine with it :wink: 

I'll see what I can do.